dumper/dataWriter.go

```diff
@@ -346,6 +433,19 @@ func formatOnConflict(row []sqlUtil.RowDataStructure, table schemareader.Table)
 		} else {
 			return "(version, release, epoch, ((evr).type)) WHERE epoch IS NOT NULL DO NOTHING"
 		}
+
+	case "rhnpackagecapability":
```

Comment:
Why is this in here? are we exporting this table for configuration channels?

Reply:
I've no idea, really. I'll just remove it since we do not  have it on master and it just works here even without this case.